classes/admin/class-kp-form-fields.php

```diff
@@ -179,7 +179,7 @@ private static function kp_form_production_username() {
 			'default'           => '',
 			'desc_tip'          => false,
 			'custom_attributes' => array(
-				'autocomplete' => 'off',
```

Comment:
Where did you get this from? Since we have the same logic in KCO right now, that solved an identical issue before https://github.com/krokedil/klarna-checkout-for-woocommerce/blob/master/classes/class-kco-fields.php#L101-L103

Im not sure this is the correct way to do this, so id like to know where you got it from.

Reply:
I am not sure why this is not an issue in KCO, but in KP, `autocomplete=off` is being ignored. For some reason, Chrome will always fill the Australian production MID with the admin credentials.